lib/perl/Genome/Model/Tools/Bsmap/MethCalcConversionRate.pm

```diff
@@ -0,0 +1,263 @@
+# % Last Change: Thu Jan 15 01:21:18 PM 2015 CST
+package Genome::Model::Tools::Bsmap::MethCalcConversionRate;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Model::Tools::Bsmap::MethCalcConversionRate {
+    is => 'Command',
+    has => [
+        snvs_file => {
+            is => 'String',
+            is_optional => 1,
+            doc => 'Use snvs.hq file to calculate methylation conversion',
+        },
+        model_id => {
```

Comment:
The command operations will need to run should go under Genome::Model::ReferenceAlignment::Command.  There are currently a handful of QC commands located there.  All of the tool can be moved or you could refactor the tool to operate on files alone and the G:M:RA:Command would call the tool after gathering the required file inputs.  There are several QC tools that would benefit from automation.  For now, the command should work to serve operations needs.